src/components/presentations/AdminTab/admintab.scss

```diff
@@ -40,7 +40,7 @@
             padding: 15px;
             outline: none;
             background: $primary-color;
-            color: #fff;
+            color: white;
```

Comment:
Good job @amaechi-chuks, any reason why are you using `white` instead of the `hex` value?